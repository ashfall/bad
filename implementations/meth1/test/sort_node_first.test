#!/bin/sh

mkdir -p ${srcdir}/.test-tmp
rm -f ${srcdir}/.test-tmp/sort_node_first.out

${srcdir}/app/meth1_node_test_first \
  ${srcdir}/test/in.s0000.e2000.recs \
  | grep "first:" \
  > ${srcdir}/.test-tmp/sort_node_first.out

diff \
  ${srcdir}/test/sort_node_first.out \
  ${srcdir}/.test-tmp/sort_node_first.out

