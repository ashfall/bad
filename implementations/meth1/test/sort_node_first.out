first: 002f43f56cedf802a836
